tools/Modules/RFC/RFC.psd1

```diff
@@ -0,0 +1,130 @@
+#
+# Module manifest for module 'RFC'
+#
+# Generated by: jimtru
+#
+# Generated on: 4/22/19
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'RFC.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = 'e05ba184-7bd8-4a16-9f41-93d7c7bd4ee6'
+
+# Author of this module
+Author = 'jimtru'
+
+# Company or vendor of this module
+CompanyName = 'Unknown'
```

Comment:
This should be Microsoft.  Can we remove all the comments/commented out members?

Reply:
fixed